src/file.cr

```diff
@@ -338,6 +338,19 @@ class File < IO::FileDescriptor
   # *dir* is given, in which case it will be used as the starting point.
   #
   # ```
+  # File.absolute_path("foo")             # => "/home/.../foo"
+  # File.absolute_path("baz", "/foo/bar") # => "/foo/bar/baz"
+  # ```
+  def self.absolute_path(path, dir = nil) : String
```

Comment:
*If* we have reached a consensus that the `File...` methods will be going away in favour of `Path` then perhaps we shouldn't add this.